Integrations/python/deephaven/java/__init__.py

```diff
@@ -0,0 +1,190 @@
+#
+# Copyright (c) 2016 - 2021 Deephaven Data Labs and Patent pending
+#
+"""
+A deephaven submodule for accessing java types from Python
+"""
+
+import jpy
+import wrapt
+import sys
+
+# None until the first _defineSymbols() call
+Runtime = None
+
+#
+# Define Java types
+# Java types from Python are represented as jpy wrappers for the corresponding
+# Java type object.
+# 
+
+# None until the first _defineSymbols() call
+# java.io
+File = None
+# java.lang
+"""Boolean class: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/Boolean.html"""
```

Comment:
I'm not sure I'm a fan of direct links to javadocs - I think doing:

```
Java "java.lang.Boolean" type.
```
or something similar is better.

Reply:
People want links to click on.  Otherwise they are having to cut and paste to read the docs.